kinto/config/kinto.tpl

```diff
@@ -36,6 +36,11 @@ multiauth.policies = basicauth
 # kinto.experimental_collection_schema_validation = true
 
 #
+# Plugins
+#
+kinto.includes = kinto.plugins.default_bucket
```

Comment:
So that adding it automatically could be removed in 2.X, whereas config files can stay